characteristic_read/index.html

```diff
@@ -0,0 +1,67 @@
+<!DOCTYPE html>
+<!--
+ Copyright 2020 Google LLC
+
+ Licensed under the Apache License, Version 2.0 (the "License");
+ you may not use this file except in compliance with the License.
+ You may obtain a copy of the License at
+
+      http://www.apache.org/licenses/LICENSE-2.0
+
+ Unless required by applicable law or agreed to in writing, software
+ distributed under the License is distributed on an "AS IS" BASIS,
+ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ See the License for the specific language governing permissions and
+ limitations under the License.
+-->
+
+<html>
+
+<head>
+  <meta charset="UTF-8">
+  <title>Bluetooth Read-only Characteristic</title>
+  <script src="../shared/shared.js"></script>
+  <script src="web_app.js"></script>
+  <link rel="stylesheet" href="../css/main.css">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+</head>
+
+<body onload="init()">
+  <h1>Read-only Characteristic</h1>
+  <p>This is a simple test to read and display a single characteristic value.
+  </p>
+  <div id="have_bluetooth" style="visibility: visible;">
+    <h2>Step 1</h2>
+    <div class="container">
+      <div class="flex">
+        <div class="inRow">
+          Press the button to load the code to the Espruino IDE.
+          From there flash any Bluetooth capable Espruino device.
+        </div>
+        <div class="inRow">
+          <button type="button" onclick="loadCode()">Load Device Code</button>
+        </div>
+      </div>
+    </div>
+
+    <h2>Step 2</h2>
+    <div class="container">
+      <div class="flex">
+        <div class="inRow">
+          Once running, start the test.
+        </div>
+        <div class="inRow">
+          <button id="btn_device_connect" type="button" onclick="startTest()">Start Test</button>
+        </div>
+      </div>
+    </div>
+
+    <p>Status:</p>
+    <pre id='status'></pre>
+  </div>
+  <div id="no_bluetooth" style="visibility: collapse;">
```

Comment:
Since this isn't a table use "hidden" instead of "collapse".

Reply:
done.